sickbeard/manual_search.py

```diff
@@ -211,20 +211,21 @@ def get_provider_cache_results(indexer, show_all_results=None, perform_search=No
     # Always start a search when no items found in cache
     if not found_items or int(perform_search):
         # retrieve the episode object and fail if we can't get one
-        ep_obj = getEpisode(show, season, episode)
-        if isinstance(ep_obj, str):
-            # ui.notifications.error(u"Something went wrong when starting the manual search for show {0}, and episode: {1}x{2}".
-            # format(show_obj.name, season, episode))
-            provider_results['error'] = 'Something went wrong when starting the manual search for show {0}, \
-            and episode: {1}x{2}'.format(show_obj.name, season, episode)
-
-        # make a queue item for it and put it on the queue
-        ep_queue_item = search_queue.ForcedSearchQueueItem(ep_obj.show, ep_obj, bool(int(down_cur_quality)), True)  # pylint: disable=maybe-no-member
-
-        sickbeard.searchQueueScheduler.action.add_item(ep_queue_item)
-
-        # give the CPU a break and some time to start the queue
-        time.sleep(cpu_presets[sickbeard.CPU_PRESET])
+        if episode:
+            ep_obj = getEpisode(show, season, episode)
```

Comment:
@p0psicles need to figure it out how to season search. I guess we need to pass multiple episodes